backend/api.py

```
@@ -12,14 +12,51 @@
 from backend.routes.diagnostics.diagnostics_router import diagnostics_router
 from backend.routes.institution.institution_router import institution_router
 from backend.routes.user.user_router import user_router
+from sqlmodel import Session, create_engine, text
+
+from backend.models_api import ResponseModel
```

Ruff (0.12.2)

Redefinition of unused `ResponseModel` from line 7

Remove definition: `ResponseModel`

(F811)